packages/applicant/next.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   },
   publicRuntimeConfig: {
     subPath: process.env.SUB_PATH || '/apply/applicant',
+    oneLoginEnabled: process.env.ONE_LOGIN_ENABLED,
```

Comment:
If its not already need to add a default value to the .env otherwise it wont get pulled up to ECS